src/niveristand/legacy/NIVeriStand.py

```diff
@@ -28,7 +28,7 @@
 _internal.dummy()
 warnings.warn(
     "NIVeriStand.py module is deprecated. "
-    "Use only if required functionality is not yet present in niveristand.clientapi",
+    "Ashu 123 Use only if required functionality is not yet present in niveristand.clientapi",
```

Comment:
was this intentional ?

Reply:
I was debugging. do't knw when git added the reviewers. :D